modules/olm-operator-maturity-model.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * operators/olm-what-operators-are.adoc
+// * operators/understanding/olm-what-operators-are.adoc
```

Comment:
operator-maturity-model.png is not rendering correctly in the preview. Might need to move that image over?

Reply:
Oops, meant to fix some of these broken images due to file moves.

Reply:
Missing symlinks added! \o/